azure-pipelines.yml

```diff
@@ -42,152 +27,183 @@ variables:
   Artifacts: $(Build.SourcesDirectory)/artifacts/
   VstsArtifacts: "$(Artifacts)"
   DOTNET_SKIP_FIRST_TIME_EXPERIENCE: "true"
+  IsMaster: $[eq(variables['Build.SourceBranch'], 'refs/heads/master')]
+  IsRelease: $[startsWith(variables['Build.SourceBranch'], 'refs/tags/v')]
+  IsBetaRelease: $[and(startsWith(variables['Build.SourceBranch'], 'refs/tags/v'), contains(variables['Build.SourceBranch'], '-beta.'))]
+  IsOfficialRelease: $[and(startsWith(variables['Build.SourceBranch'], 'refs/tags/v'), not(contains(variables['Build.SourceBranch'], '-beta.')))]
+
+stages:
+  - stage: CreateBetaReleaseTag
+    displayName: "Tag Beta Release"
```

Comment:
Running against master branch no longer builds artifacts directly. Instead it creates a draft release and tag which will then cause this pipeline to build a prerelease.